### ISSUE TITLE
Druids starving

### ISSUE THREAD

**Issue by Shamshalabim** [TangoTek/TekTopia-Community#242]
**Describe the bug**
On many occasions now we've had a druid continue working/sleeping, etc until they starve to death. It's not a shortage of food.

**To Reproduce**
1. Get druid
2. Wait for druid to die of starvation

**Expected behavior**
I'd expect the druids to eat like everyone else.

** Server/Location **
Main 1
X: -3716, Y: 72, Z: 2019

**Comment by Sobertron** [comment]
There have been multiple players reporting this in game. Would have reported this days ago but someone said they already had.

**Comment by JohnyDL** [comment]
Duplicate of #164

**Comment by Shamshalabim** [comment]
Yes, duplicate of 164, but the comments for that one state it as closed..

**Comment by Shamshalabim** [comment]
.. as the guy figured out it was because his food was out..

**Comment by JohnyDL** [comment]
there's a big open/closed at the top of each issue, he re-opened it

**Comment by bhdavies** [comment]
The Druid is a vegetarian, and will work them selves to starvation. They will fill up a mine and that takes so much hunger a carrot will bring them just high enough to cast again then the Druid will die of starvation before they can path find to the storage room out of the mines.

**Comment by Apocangel** [comment]
Was about to report this same thing. My druids keep working themselves to death. Poor dudes were just a few paces away from the storage when they kicked it...

**Comment by BlkHellfire** [comment]
I just lost a druid to starvation. There was food in storage.

**Comment by Shamshalabim** [comment]
I just lost a druid to starvation. He stood there in the middle of the mineshaft tunnel and starved, 1 damage at a time. When he died he dropped a carrot, so he had food. When I got to him he only had 3 health left.. I nudged him and he started earth reform, and then died half way thro the cast. It was daytime.

**Comment by Creenel** [comment]
Happens to me too!

**Comment by Nightshae** [comment]
Lost two of them tonight, and had another 5 or 6 nearly starving to death before they got to the storeroom and reached food.  All of them were down in the mines.  This caught me by surprise, I hadn't seen it happen for quite a while.

**Comment by Apocangel** [comment]
A quick workaround while tango fixes this is to have the mines just below ground level. The druid's earth regen works as if within diamond level regardless of the actual location of the mine. When they regrow, they generate all ores. So, whenever they work themselves to starvation, they can run to the storage just a few blocks away instead of climbing a huge flight of stairs. 

Haven't lost a single one since I started doing this.

**Comment by Nightshae** [comment]
Good thought.  I'll see about moving my mines up and blocking the lower levels until an update.  Thank you.

    On Tuesday, September 17, 2019, 11:59:37 PM AKDT, Apocangel <notifications@github.com> wrote:  
 
 
A quick workaround while tango fixes this is to have the mines just below ground level. The druid's earth regen works as if within diamond level regardless of the actual location of the mine. When they regrow, they generate all ores. So, whenever they work themselves to starvation, they can run to the storage just a few blocks away instead of climbing a huge flight of stairs.

Haven't lost a single one since I started doing this.
